src/constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
+ * Constant for Icon center position offset in pixels
+ */
+export const ICON_CENTER_POSITION = 12;
```

Comment:
If these constants are related to datagird  filter then please move them in their respective file